backend/src/schema/resolvers/searches.js

```diff
@@ -40,8 +40,7 @@ export default {
       YIELD node as resource, score
       MATCH (resource)
       WHERE score >= 0.5
-      AND NOT (resource.deleted = true OR resource.disabled = true
-      OR (:User { id: $thisUserId })-[:BLOCKED]-(resource))
+      AND NOT (resource.deleted = true OR resource.disabled = true)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:05:08Z" title="Thursday, January 23rd 2020, 6:05:08 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -40,8 +39,7 @@ export default {
       YIELD node as resource, score
       MATCH (resource)
       WHERE score >= 0.5
-      AND NOT (resource.deleted = true OR resource.disabled = true
-      OR (:User { id: $thisUserId })-[:BLOCKED]-(resource))
+      AND NOT (resource.deleted = true OR resource.disabled = true)
```


I think this introduced a bug: Your search results would contain posts by muted users.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:42:04Z" title="Thursday, January 23rd 2020, 6:42:04 pm +01:00">Jan 23, 2020</time>_

---


this is part of the `UserQuery`, so it would so users who you have blocked... this was specifically asked for.
I have updated the `PostQuery` to remove posts by those you have `Muted`... and found out why it was till passing cypress...